backend/tests/Feature/BomTest.php

```
@@ -189,6 +189,98 @@ public function test_admin_can_remove_bom_item(): void
         $this->assertSoftDeleted('bom_items', ['id' => $bomItem->id]);
     }
 
+    // ── Product-type components (sub-assemblies) ────────────────
+
+    public function test_admin_can_add_product_type_to_bom(): void
+    {
+        $productType = ProductType::factory()->create();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\ProductType::factory().


(staticMethod.notFound)